src/test/java/com/artipie/rpm/misc/StorageLockerTest.java

```diff
@@ -82,7 +83,7 @@ void addsLockAndThrowsExceptionOnSecondLock() {
             ).getCause().getMessage(),
             new StringContains(
                 String.format(
-                    "Repository %s is already being updated, %d locks found.", key.string(), 1
+                    "Repository '%s' is already being updated, %d locks found.", key.string(), 1
```

Comment:
@olenagerasimova instead of specifying the number of locks as a literal param, just specify it in the string literal.